assets/lang/pt_BR.ini

```diff
@@ -1,1218 +1,1219 @@
+Warning: Video memory FULL, switching to slow caching mode = AVISO: Memória de vídeo CHEIA, trocando pro modo de cache lento.
+ZIP file detected (Require UnRAR) = O arquivo está comprimido (ZIP).\nPor favor descomprima-o primeiro (tente o UnRAR).
+ZIP file detected (Require WINRAR) = O arquivo está comprimido (ZIP).\nPor favor descomprima-o primeiro (tente o WinRAR).
+
+[Game]
+Asia = Ásia
+Calculate CRC = Calcular CRC
+ConfirmDelete = Apagar
+Create Game Config = Criar configuração do jogo
+Create Shortcut = Criar atalho
+Delete Game = Apagar jogo
+Delete Game Config = Apagar configuração do jogo
+Delete Save Data = Apagar dados do save
+Europe = Europa
+Game = Jogo
+Game Settings = Configurações do jogo
+Homebrew = Homebrew
+Hong Kong = Hong Kong
+InstallData = Dados da instalação
+Japan = Japão
+Korea = Coréia
+MB = MBs
+One moment please... = Um momento por favor...
+Play = Jogar
+Remove From Recent = Remover dos "Recentes"
+SaveData = Dados do save
+Setting Background = Configurando o cenário de fundo
+Show In Folder = Mostrar na pasta
+USA = EUA
+Use UI background = Usar cenário de fundo da interface do usuário
+
+[Graphics]
+% of the void = % do vazio
+% of viewport = % do visor
+%, 0:unlimited = %, 0 = ilimitado
+(supersampling) = (super amostragem)
+(upscaling) = (ampliação)
+1x PSP = 1× PSP
+2x = 2×
+2x PSP = 2× PSP
+3x = 3×
+3x PSP = 3× PSP
+4x = 4×
+4x PSP = 4× PSP
+5x = 5×
+5x PSP = 5× PSP
+6x PSP = 6× PSP
+7x PSP = 7× PSP
+8x = 8×
+8x PSP = 8× PSP
+9x PSP = 9× PSP
+10x PSP = 10× PSP
+16x = 16×
+Aggressive = Agressivo
+Alternative Speed = Velocidade alternativa (em %, 0 = ilimitada)
+Alternative Speed 2 = Velocidade alternativa 2 (em %, 0 = ilimitada)
+Anisotropic Filtering = Filtragem anisotrópica
+Auto = Auto
+Auto (1:1) = Auto (1:1)
+Auto (same as Rendering) = Auto (a mesma que a da resolução de renderização)
+Auto FrameSkip = Auto-pulo dos frames
+Auto Max Quality = Auto-Maximizar a Qualidade
+Auto Scaling = Auto-redimensionamento
+Backend = Backend
+Balanced = Balanceado
+Bicubic = Bi-cúbico
+BlockTransfer Tip = Alguns jogos requerem que isto esteja Ligado pros gráficos estarem corretos
+BlockTransferRequired = Aviso: Esse jogo requer que a "simulação dos efeitos de transferência dos blocos" esteja definida como Ligada.
+Both = Ambos
+Buffer graphics commands (faster, input lag) = Buffer dos comandos dos gráficos (mais rápido, atraso na entrada dos dados)
+Buffered Rendering = Renderização com buffer
+BufferedRenderingRequired = Aviso: Este jogo requer que o "modo de renderização" esteja definido "com buffer".
+Camera = Câmera
+Camera Device = Dispositivo de câmera
+Cardboard Screen Size = Tamanho da tela	(em % do visor)
+Cardboard Screen X Shift = Deslocamento do X (em % do espaço vazio)
+Cardboard Screen Y Shift = Deslocamento do Y (em % do espaço vazio)
+Cardboard VR Settings = Configurações do Google VR Cardboard

```

Comment:
VR Cardboard have little to do with Google I think

Reply:
That's strange, the word google is in the language file of latest unofficial build in the site.